### ISSUE TITLE
TypeError: must be real number, not str

### ISSUE THREAD

**Issue by andrewkho** [IntelPython/daal4py#564]
I'm trying to get daal4py to work to improve prediction speed but having some trouble with LightGBM

**Describe the bug**
When calling get_gbt_model_from_lightgbm on a LightGBM model, I get the error:
```
TypeError                                 Traceback (most recent call last)
<ipython-input-13-a87d37845ba5> in <module>
----> 1 daal_model = d4p.get_gbt_model_from_lightgbm(m.lgb_model)

build/daal4py_cy.pyx in _daal4py.get_gbt_model_from_lightgbm()

build/daal4py_cy.pyx in _daal4py.gbt_regression_model_builder.add_split()

TypeError: must be real number, not str
```

**To Reproduce**
1. Train LightGBM model 
2.
```
import daal4py as d4p
daal_model = d4p.get_gbt_model_from_lightgbm(lgb_model)
```
3. see TypeError

**Expected behavior**

**Output/Screenshots**
If applicable, add output/screenshots to help explain your problem.

**Environment:**
 - OS: OSX 10.15.7
 - Compiler: n/a
 - Version: lightgbm==3.1.1 daal4py==2021.1

**Comment by andrewkho** [comment]
It looks like this happens when the lightGBM model is trained using a Dataset created with `categorical_feature` param.

**Comment by andrewkho** [comment]
If this is the intended behaviour, would be nice to have a better error message!

**Comment by RukhovichIV** [comment]
Thank you for reporting this bug. daal4py unlike LightGBM doesn't currently support categorical feature handling in its models. But nothing was reported when such models were passed to it.
A quick fix PR (#566) has been created, so the behaviour will change soon.

**Comment by andrewkho** [comment]
TY!!
